Source/WebCore/loader/FetchOptions.h

```diff
@@ -64,14 +64,11 @@ struct FetchOptions {
     ReferrerPolicy referrerPolicy { ReferrerPolicy::EmptyString };
     bool keepAlive { false };
     String integrity;
-    // FIXME: These std::optional<ScriptExecutionContextIdentifier> waste 15 bytes each.
-    // Identifier of https://fetch.spec.whatwg.org/#concept-request-client
-    std::optional<ScriptExecutionContextIdentifier> clientIdentifier;
-    // Identifier of https://fetch.spec.whatwg.org/#concept-request-reserved-client
-    std::optional<ScriptExecutionContextIdentifier> resultingClientIdentifier;
+    Markable<UUID> clientIdentifier; // Identifier of https://fetch.spec.whatwg.org/#concept-request-client
+    Markable<UUID> resultingClientIdentifier; // Identifier of https://fetch.spec.whatwg.org/#concept-request-reserved-client
```

Comment:
We can probably remove this one from FetchOptions as well.
clientIdentifier might be a bit more work but we should probably do it as well.
This would allow to reunify persistent FetchOptions from non persistent FetchOptions.